### PR TITLE
Show a dialog to resolve conflicts when transferring an item and the destination already exits

### LINKED ISSUE #1893: Support overwrite experience during paste operation
**Is your feature request related to a problem? Please describe.**
When you're pasting a file to a folder that already has a file with the same name, there is no choice to overwrite it - the app always creates a copy.

**Describe the solution you'd like**
Offer a dialog with the choice of either overwriting or creating a copy.

**Additional context**
App version: v0.14.0

### PR DESCRIPTION
**Pull request by gave92**
Fixes #1893

This PR adds a dialog (identical to the one already existing for the rename operation) when moving/copying an item and the destination already exists.
Dialog options: Generate new name, Overwrite, Do nothing (escape key)
This is for both files and folders, there is no "merge folders" option

In addition:
- enables the dialog shown when pasting a folder inside itself
- fixes an issue for which a banner was left in the status center after canceling a delete operation
- fixes an issue for which the app crashed when pressing delete key -> [2010984911u](https://appcenter.ms/orgs/Files-UWP/apps/Files/crashes/errors/2010984911u/overview)

### CODE REVIEW

File: Files/App.xaml.cs [hunk 1]
```diff
@@ -39,10 +39,7 @@ sealed partial class App : Application
     {
         private static bool ShowErrorNotification = false;
 
-        private readonly static CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
-        public static CancellationToken CancellationToken = cancellationTokenSource.Token;
```
Comment (d2dyno1) [inline Files/App.xaml.cs:43]: :(

**Comment by gave92** [comment]
> :(
I simply moved it to ModernShellPage :)
It's not used atm, right? In the future we could use it to stop a particular operation, but would not make sense for it to be global i guess

**Comment by tsvietOK** [comment]
@gave92 Maybe i am doing something wrong, but when i copying file or folder and pasting in the same place, it shows dialog with 2 buttons instead of 3.

**Comment by gave92** [comment]
@tsvietOK that's expected, currently the only way to cancel without doing anything is to press the escape key
You're right we need to add the "CloseButtonText" to both this and the rename dialog

**Comment by tsvietOK** [comment]
@gave92 Oh, ok. Sorry, my fault. Would you want to add this button in this or another PR?

**Comment by gave92** [comment]
@yair100 do you think I should add a third button to the rename and copy dialogs to cancel the operation?
Or is pressing the escape key to cancel sufficient?

**Comment by yair100** [comment]
> @yair100 do you think I should add a third button to the rename and copy dialogs to cancel the operation?
> Or is pressing the escape key to cancel sufficient?

A cancel button is especially important for tablet users so we should probably add one.

**Comment by gave92** [comment]
Cancel button added to dialog 👍

**Review by yair100 — APPROVED** [review]

**Pull request merged**